main/src/library/MutList.flix

```diff
@@ -108,14 +82,6 @@ namespace MutList {
     ///
     pub def memberOf(x: a, v: MutList[a]): Bool & Impure with Eq[a] = exists(y -> y == x, v)
 
-    ///
-    /// Optionally finds the smallest element of `v` according to the `Order` on `a`.
-    ///
-    /// Returns `None` if `v` is empty.
-    ///
-    pub def minimum(v: MutList[a]): Option[a] & Impure with Order[a] =
```

Comment:
For example, even though the functor instance is removed, we should still have a minimum function.

Reply:
I accidentally merged with the other PR branch #2704, meaning it deleted these functions again, and it won't let me revert...